### PR TITLE
Yieldmo Bid Adapter: add support for the floors module

### PR DESCRIPTION
**Pull request by ym-dlabuzov**
## Type of change

- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

Support for the floors module added.

adapter maintainer email: opensource@yieldmo.com

- [x] official adapter submission

## Other information

Related to #6465

### CODE REVIEW

File: modules/yieldmoBidAdapter.js [hunk 1]
```diff
@@ -304,6 +304,16 @@ function openRtbImpression(bidRequest) {
   return imp;
 }
 
+function getBidFloor(bidRequest) {
+  let floorInfo = {};
+
+  if (typeof bidRequest.getFloor === 'function') {
+    floorInfo = bidRequest.getFloor({ currency: CURRENCY, mediaType: VIDEO, size: '*' });
```
Comment (bszekely1) [inline modules/yieldmoBidAdapter.js:311]: According to your bid parameters, you guys support bidfloor. It seems you are checking for video in the openrtb object for video but not support floors for banners. Is this intended?

Comment (ym-dlabuzov) [reply modules/yieldmoBidAdapter.js]: Thanks, this was a bug, fixed now.

**Comment by ym-dlabuzov** [comment]
Interesting, looks like it's not our test failed...

1) should build utm data from local storage
     Prebid Manager Analytics Adapter build utm tag data
     Timeout of 2000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.

**Review by patmmccann — APPROVED** [review]

**Comment by patmmccann** [comment]
Looks good now, ty!

File: modules/yieldmoBidAdapter.js [hunk 2]
```diff
@@ -312,6 +313,16 @@ function openRtbImpression(bidRequest) {
...
 }
 
+function getBidFloor(bidRequest, mediaType) {
+  let floorInfo = {};
+
+  if (typeof bidRequest.getFloor === 'function') {
+    floorInfo = bidRequest.getFloor({ currency: CURRENCY, mediaType, size: '*' });
+  }
+
+  return floorInfo.floor || bidRequest.params.bidfloor || bidRequest.params.bidFloor || 0;
```
Comment (bszekely1) [inline modules/yieldmoBidAdapter.js:323]: you duplicated your check on bidRequest.params.bidFloor. Should not cause an error, however it's unnecessary.

Comment (ym-dlabuzov) [reply modules/yieldmoBidAdapter.js:323]: It's `bidfloor` vs `bidFloor`, they are a bit different

Comment (bszekely1) [reply modules/yieldmoBidAdapter.js:323]: Got it. I didn't catch the case.

**Pull request merged**